src/views/BasicPage.vue

```diff
@@ -14,14 +14,51 @@
 				required: true
 			}
 		},
+		data() {
+			return {
+				paddingLeft: 0, 
+			}
+		},
```

Comment:
as before, consistency
```suggestion
		},
		
```